README.md

```diff
@@ -45,51 +45,48 @@ data science with BigQuery.
 
-* `my_project`: This is your project name that contains the BigQuery dataset.
-* `gs://my_bucket/vcffiles/*.vcf`: A location in Google Cloud Storage where the
+* `PROJECT_ID`: This is your project ID that contains the BigQuery dataset.
+* `INPUT_PATTERN`: A location in Google Cloud Storage where the
   VCF file are stored. You may specify a single file or provide a pattern to
   load multiple files at once. Please refer to the
   [Variant Merging](docs/variant_merging.md) documentation if you want
   to merge samples across files. The pipeline supports gzip, bzip, and
   uncompressed VCF formats. However, it runs slower for compressed files as they
   cannot be sharded.
-* `my_bigquery_dataset`: Your BigQuery dataset to store the output.
-* `my_bigquery_table`: This can be any ID you like (e.g. vcf_test).
-* `gs://my_bucket/staging` and `gs://my_bucket/temp`: These can be any folder in
-  Google Cloud Storage that your project has write access to. These are used to
-  store temporary files needed for running the pipeline.
-
-```yaml
-name: vcf-to-bigquery-pipeline
-docker:
-  imageName: gcr.io/gcp-variant-transforms/gcp-variant-transforms
-  cmd: |
-    ./opt/gcp_variant_transforms/bin/vcf_to_bq \
-      --project my_project \
-      --input_pattern gs://my_bucket/vcffiles/*.vcf \
-      --output_table my_project:my_bigquery_dataset.my_bigquery_table \
-      --staging_location gs://my_bucket/staging \
-      --temp_location gs://my_bucket/temp \
-      --job_name vcf-to-bigquery \
-      --runner DataflowRunner
-```
-
-Next, run the following command to launch the pipeline. Replace `my_project`
-with your project name, `gs://my_bucket/temp/runner_logs` with a Cloud Storage
-folder to store the logs from the pipeline.
+* `OUTPUT_TABLE`: The full path to a BigQuery table to store the output.
+* `TEMP_LOCATION`: This can be any folder in Google Cloud Storage that your
+  project has write access to. It's used to store temporary files and logs
+  from the pipeline.
 
 ```bash
+#!/bin/bash
+# Parameters to replace:
+PROJECT_ID=PROJECT_ID
+INPUT_PATTERN=gs://BUCKET/*.vcf
+OUTPUT_TABLE=PROJECT_ID:BIGQUERY_DATASET.BIGQUERY_TABLE
+TEMP_LOCATION=gs://BUCKET/temp
+
+COMMAND="/opt/gcp_variant_transforms/bin/vcf_to_bq \
+  --project ${PROJECT_ID} \
+  --input_pattern ${INPUT_PATTERN} \
+  --output_table ${OUTPUT_TABLE} \
+  --temp_location ${TEMP_LOCATION} \
+  --job_name vcf-to-bigquery \
+  --runner DataflowRunner"
 gcloud alpha genomics pipelines run \
-    --project my_project \
-    --pipeline-file vcf_to_bigquery.yaml \
-    --logging gs://my_bucket/temp/runner_logs \
-    --zones us-west1-b \
-    --service-account-scopes https://www.googleapis.com/auth/bigquery
+  --project "${PROJECT_ID}" \
+  --logging "${TEMP_LOCATION}/runner_logs_`date +%Y%m%d_%H%M%S`.log" \
+  --zones us-west1-b \
+  --service-account-scopes \
+    https://www.googleapis.com/auth/bigquery,https://www.googleapis.com/auth/cloud-platform \
```

Comment:
Does BigQuery specifically require the bigquery scope or is cloud-platform sufficient? (Services are migrating to all use cloud-platform eventually) [Also, I don't assume the short scope works here? eg just cloud-platform]